src/core/unittest/VersionNegExtTest.cpp

```diff
@@ -25,7 +29,11 @@ TEST(VersionNegExtTest, ParseVersionInfoFail)
     };
 
     QUIC_VERSION_INFORMATION_V1 ParsedVI = {0};
-    QUIC_CONNECTION* NoOpConnection = (QUIC_CONNECTION*)0x1;
+    struct { QUIC_HANDLE Handle;
+        QUIC_CONNECTION Connection;
+    } Connection {};
```

Comment:
I don't understand this struct. Is it supposed to be union?

Reply:
It's not supposed to be a union, but I see now that QUIC_CONNECTION includes the QUIC_HANDLE, so I'll remove the struct and just use a regular QUIC_CONNECTION struct on the stack.